### PR TITLE
Bring sysinternals changes from Extras bucket

### PR DESCRIPTION
**Pull request by Bibz87**
## What type of Apps related PR is this? (check all applicable)

- [ ] ✨ Add Manifest
- [ ] ♻️ Refactor
- [X] 🐛 Bug Fix
- [ ] 👷 Optimization
- [ ] 🚩 Other

### Add/Modify a description in the manifest?

- [X] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed

### Check URLs?

- [X] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed

### Update to the latest version?

- [X] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed

### Update Hashes?

- [X] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed

### Format the json manifest?

- [X] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed

### Notes in the manifest?

- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [X] 🙅 no, because they aren't needed

### Application Dependencies?

- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [X] 🙅 no, because they aren't needed

### Application Suggestions?

- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [X] 🙅 no, because they aren't needed

### Pre-installation steps to perform?

- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [X] 🙅 no, because they aren't needed

### Post-installation steps to perform?

- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [X] 🙅 no, because they aren't needed

### File persistence to perform?

- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [X] 🙅 no, because they aren't needed

## [optional] What gif/emoji best describes this PR or how it makes you feel?

🐛

### CODE REVIEW

**Comment by BotServiceAutomation** [comment]
All changes look good.

Wait for review from human collaborators.
### sysinternals

- [x] Description
- [x] License
- [x] Hashes
- [x] Checkver
- [x] Autoupdate

**Review by alexchapar — APPROVED** [review]

**Pull request merged**
